lib/ansible/module_utils/fortios.py

```diff
@@ -42,6 +44,14 @@
 except ImportError:
     HAS_PYFG = False
```

Comment:
Why aren't we able to continue forward using `pyFG` as opposed to writing all the interactions ourselves?

Reply:
Because pyFG is using SSH scraping and I wanted to use API, as it is a better way of managing devices.